sources/operator.1.12.0.pxc-operator.dep.json

```diff
@@ -0,0 +1,32 @@
+{
+  "backup": {
+    "8.0.29": {
+      ">=": [
+        {
+          "var": "productVersion"
+        },
+        "8.0"
+      ]
+    },
+    "2.4.26-1": {
```

Comment:
Seems 2.4.26 is official version: https://docs.percona.com/percona-xtrabackup/2.4/release-notes/2.4/2.4.26.html